client/deposits/strings.ts

```diff
@@ -32,3 +32,106 @@ export const depositStatusLabels: Record<
 	canceled: __( 'Canceled', 'woocommerce-payments' ),
 	failed: __( 'Failed', 'woocommerce-payments' ),
 };
+
+/**
+ * Mapping of payout failure code to display string.
+ */
+export const payoutFailureMessages: Record< PayoutFailureCode, string > = {
+	insufficient_funds: __(
+		'Your account has insufficient funds to cover the transfer.',
+		'woocommerce-payments'
+	),
+	bank_account_restricted: __(
+		'The bank account has restrictions on either the type or number of transfers allowed. This normally indicates that the bank account is a savings or other non-checking account.',
+		'woocommerce-payments'
+	),
+	debit_not_authorized: __(
+		'Debit transactions are not approved on the destination bank account. Bank accounts need to be set up for both credit and debit transfers.',
+		'woocommerce-payments'
+	),
+	invalid_card: __(
+		'The card used was invalid. This usually means the card number is invalid or the account has been closed. Please verify card details before retrying.',
```

Comment:
I ran through these sentences to AI and this one can be improved:

```suggestion
		'The card used was invalid. This usually means the card number is incorrect or the account has been closed. Please verify the card details before retrying.',
```

Reply:
I have used these strings from what was sent by Stripe. The suggestions noted by you are good and I will change them.

Reply:
Changed here, thanks! - [d256e22](https://github.com/Automattic/woocommerce-payments/pull/10462/commits/d256e22ea41da96714e462291a2032cd16427eb0)

Reply:
> I have used these strings from what was sent by Stripe.

Ah. I didn't know that. That should be safe to use then but it's OK to have something different as well, part of owning our own product.